src/webportal/src/app/job/job-view/fabric/JobList/Filter.js

```diff
@@ -62,6 +70,16 @@ class Filter {
     if (users && users.size > 0) {
       query.username = Array.from(users).join(',');
     }
+    if (priorities && priorities.size > 0) {
+      query.jobPriority = Array.from(priorities).map(priority => {
+        switch (priority) {
+          case 'Opportunistic': return 'oppo';
+          case 'Product': return 'prod';
```

Comment:
there is some lint error could be fixed by prettier

Reply:
Done.